style.css

```diff
@@ -0,0 +1,72 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  position: sticky;
+}
+
+#logo {
+  color: #6070ff;
+  text-decoration: none;
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+.menu {
+  color: #6070ff;
+  font-size: 1.3rem;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  background-image: url("./images/header-shapes.png");
+  background-color: #fff;
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.greetings {
+  color: black;
+  font-size: 4.5rem;
```

Comment:
- [ ] Kindly consider changing this font-size to 40px as it is in the figma design because it is a project requirement
   -  Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Reply:
resolved